src/repart/repart.c

```diff
@@ -7487,7 +7510,42 @@ static int context_fstab(Context *context) {
                 }
         }
 
-        r = flink_tmpfile(f, t, path, 0);
+        switch (arg_append_fstab) {
+        case APPEND_AUTO:
+                fputs(AUTOMATIC_FSTAB_HEADER_END "\n", f);
+
+                r = read_full_file(path, &c, &l);
+                if (r < 0) {
+                        if (r == -ENOENT) {
+                                log_warning("File fstab not found in %s", path);
+                                break;
+                        } else
```

Comment:
you can drop the word `else` here, it carries no meaning (because the first branch has `break`). we generally want short code, void of such redundancies

Reply:
Yes, sorry.